packages/pkh-webauthn/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@didtools/pkh-webauthn",
```

```suggestion
  "name": "@didtools/key-webauthn",
```

Let's call this package `key-webuathn` since it's definitely not using the `pkh` DID.